thunx-autoconfigure/src/main/java/com/contentgrid/thunx/api/autoconfigure/AbacAutoConfiguration.java

```diff
@@ -3,13 +3,17 @@
 import com.contentgrid.thunx.spring.data.EnableAbac;
 import com.contentgrid.thunx.spring.data.querydsl.predicate.injector.rest.webmvc.SpringDataQuerydslPredicateInjectorAutoConfiguration;
 import org.springframework.boot.autoconfigure.AutoConfiguration;
-import org.springframework.boot.autoconfigure.AutoConfigureBefore;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnWebApplication.Type;
+import org.springframework.boot.autoconfigure.data.rest.RepositoryRestMvcAutoConfiguration;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
-@AutoConfiguration
+@AutoConfiguration(
+        before = SpringDataQuerydslPredicateInjectorAutoConfiguration.class,
+        after = RepositoryRestMvcAutoConfiguration.class)
```

Comment:
Nitpicking, but this looks ugly and confusing to me.

```suggestion
@AutoConfiguration(
        before = SpringDataQuerydslPredicateInjectorAutoConfiguration.class,
        after = RepositoryRestMvcAutoConfiguration.class
)
```